src/page_format_wizard.lua

```diff
@@ -89,7 +89,7 @@ end
 
 function math.round(number, round_to)
   round_to = round_to or 1
-  return math.floor(number/round_to + math.sign(number) * 0.5) * round_to
+  return math.floor((number or 0) / round_to + ((number or 0) >= 0 and 0.5 or -0.5)) * round_to
```

Comment:
Why not do it like this:

```
number = number or 0
return math.floor(number/round_to + math.sign(number) * 0.5) * round_to
```

fwiw: I don't think this code works when `number` is negative. For example:

-1.2 gets rounded to -3. (Because -1.3 - 0.5 is -2.7 and the floor of that is -3.)
-1.7 gets rounded to -2.

I think you need to use `math.ceil` as well as `-0.5` when `number` is negative. Or you could use `abs`:

```
  number = number or 0
  round_to = round_to or 1
  return math.floor(math.abs(number)/round_to + 0.5) * round_to * math.sign(number)
```